workers/gitlab_merge_request_worker/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 Augur Team
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
```

Comment:
Change this to the SPDX license identifier.

Reply:
We just shouldn’t include this file. I don’t see why we would need the whole license again below the root directory where it‘s already specified, but maybe I’m missing something.

Reply:
This might've been transferred from another repository. Ergo, the license! I guess we wouldn't require an additional one, but not sure..

Reply:
Cool. I have removed the LICENSE file.